iaso/gpkg/README.md

```diff
@@ -35,34 +35,45 @@ be sure when updating an existing one that the casing and accents matches the ta
 Note that OrgUnit and Group are linked to a particular data source but OrgUnitType are not and are shared between all projects of a tenant.
 
 Each layer table must have the following columns:
-* `name` The name of the OrgUnit
-* a geometry (can be any name as long as it's referenced as per the GeoPackage spec) [1]
+
+Required columns (must exist and cannot be null/empty/blank):
 * `ref` Reference to be used for update and linking the parent (will be in orgunit.source_ref). Must be unique.
-* `parent_ref` the reference of the parent OrgUnit may be blank or null or the `ref` of the parent (not the name). [2]
-* `group_refs` a list of groups reference to which the OrgUnit should belong. Represented as string of ref separated by the ',' character. This column is optional, delete it if you don't want to touch group affiliation. *If you keep it empty all groups will be removed from the OrgUnit when updating*. See Group table below.
+* `name` The name of the OrgUnit
 
+Optional non-nullable columns (can be missing, but if present cannot be empty):
+* `parent_ref` The reference of the parent OrgUnit. If present, must contain either:
+  - A valid `ref` of another OrgUnit
+  - The format `iaso#{db_id}` to reference an existing OrgUnit by its database ID
+  - Cannot be empty/blank if the column exists
```

Comment:
This would prevent the import of a full org unit hierarchy, since you could not import the top parent